CHANGELOG.md

```diff
@@ -1,3 +1,45 @@
+## Fleet 4.60.0 (Nov 14, 2024)
+
+### Bug fixes
+
+### Endpoint Operations
```

Comment:
```suggestion
### Observability
```

Should we use the new product categories from fleetdm.com? (aka soon-to-be product groups):

![Screenshot 2024-11-25 at 2 02 05 PM](https://github.com/user-attachments/assets/2c0ce0d7-2f8c-4b33-aa8f-c94357a0f21e)

Reply:
@noahtalerman Let's wait until we announce new product groups at beginning of Q1 to make this change.